sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/catalog/interface.scala

```diff
@@ -52,9 +52,15 @@ case class CatalogStorageFormat(
     properties: Map[String, String]) {
 
   override def toString: String = {
+    val maskedProperties = properties.map {
+      case (password, _) if password.toLowerCase == "password" => (password, "###")
```

Comment:
In Oracle, the specification of `password` and `user` in `url` does not follow the same pattern we expect. Below is an example:
```
jdbc:oracle:thin:scott/tiger@localhost:1521:orcl
```
The reference link: http://docs.oracle.com/cd/B28359_01/java.111/b31224/urls.htm

So far, the dialect does not help. Let me know whether we should add more code changes for utilizing dialects. Thanks! @rxin